src/features/MentorPage/components/MentorList/MentorCard/Expanded/Header.tsx

```diff
@@ -3,31 +3,55 @@ import type { Mentor } from '@/features/MentorPage/mentorPageApi';
 import { useGetLayoutMode } from '@/hooks/useGetLayoutMode';
 
 import ProfilePicPlaceholder from '@/static/icons/chat-profilepic.svg';
+import ProfilePicPlaceholderForMe from '@/static/icons/chat-profilepic-me.svg';
 import { breakpoints, palette } from '@/components/variables';
 import { IconButton } from '@/components/Buttons';
 import { BasicInfo } from './BasicInfo';
-import Text from '@/components/Text';
+import { Tag, type Status } from './Tag';
 import styled, { css } from 'styled-components';
 
 type Props = {
   mentor: Mentor;
+  isMe: boolean;
+  isAvailable: boolean;
+  isNew: boolean;
   onDismiss: () => void;
 };
 
-export const Header = ({ mentor, onDismiss }: Props) => {
-  const availabilityMessage = mentor.isVacationing ? 'Ei tavoitettavissa' : '';
+export const Header = ({
+  mentor,
+  isMe,
+  isAvailable,
+  isNew,
+  onDismiss,
+}: Props) => {
   const { isMobile } = useGetLayoutMode();
 
+  const getStatus = (
+    isMe: boolean,
+    isAvailable: boolean,
+    isNew: boolean,
+  ): Status => {
+    if (isMe) {
+      return 'me';
+    }
+    if (!isAvailable) {
+      return 'unavailable';
+    }
+    if (isNew) {
+      return 'new';
+    }
+    return 'empty';
+  };
```

Comment:
Instead of declaring again the `getStatus`-function, I would move this to utils/utils.ts and export from there to use in the components.